lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,73 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+
+                if ($('#accordion').length) {
```

Comment:
Can there only ever be 1 accordion on a page? It may make sense to pass in the accordion selector on init